no-computer.md

```diff
@@ -58,12 +115,20 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Everything is a DSL! Therefore the whole part of building the bike is
+designing how the person will be interacting with the device and seeing
+their thoughts and desires be expressed in real life.
```

Comment:
I agree, but I'm not totally convinced yet just from what's in your document here. I'd probably want a little more specificity (as I mentioned above)